### ISSUE TITLE
Duplicate Plots, notebook displays twice

### ISSUE THREAD

**Issue by cancan101** [statsmodels/statsmodels#1265]
I am starting IPython Notebook with `ipython notebook --pylab=inline` and then
plotting using `plot_acf`. I am currently getting the resulting plots duplicated:

![image](https://f.cloud.github.com/assets/51059/1798296/b1bb9b3a-6b3f-11e3-9b1f-27c08dfe56ed.png)

What am I doing wrong here?

Standard pandas plotting using `returns_sq.plot()` works fine (only one resulting plot).

**Comment by josef-pkt** [comment]
nothing that we are going to change here, closing

**Comment by josef-pkt** [comment]
reopening as FAQ issue for reference (but still nothing to do about it)

**Comment by JoshuaC3** [comment]
Issue still going for me. Easiest way to stop/"catch" this is:

`stld_plot = stld.plot()`

It seems to catch and hold one of the plots. Reading back through @cancan101 did state this but I'll mention it again as the easiest fix.

**Comment by lightalchemist** [comment]
It is possible to suppress the returned output from being evaluated by Jupyter simply by adding a semi-colon after the line:
`plot_acf(data);`
This will cause Jupyter to not evaluate the returned result, and hence not print the plot the second time round.

**Comment by JarnoRFB** [comment]
One way to avoid this would be to explicitly close the figure before returning it. See https://stackoverflow.com/questions/36685031/why-is-jupyter-notebook-creating-duplicate-plots-when-making-updating-plots. If the `plot_acf(data)` as the last command in a notebook cell, the figure would still get shown.

**Comment by JarnoRFB** [comment]
@josef-pkt 
You are right, closing the figure would prevent the user from showing the figure in a script. Also one might want to create several figures in a single notebook cell. Semicolons really seem to be the easiest work around.

**Comment by StevenLi-DS** [comment]
This seems happening only when the plot function is the last line of code in a cell. A side-effect of using `;` is that it will hide any text output (e.g., title, memory address, etc.) produced by matplotlib (but it shouldn't be an issue at all)
